mysql-test/columnstore/basic/t/mcs47_cpimport_central_loc_sin_source.test

```diff
@@ -25,7 +25,7 @@ CREATE TABLE t1(col1 INT, col2 INT, col3 CHAR(8)) ENGINE=Columnstore;
 --enable_result_log
 
 #Validate data
-SELECT * FROM t1;
+SELECT * FROM t1 ORDER BY col1, col3;
```

Comment:
Wouldn't it be better to use `sorted_result` pragma?

Reply:
Fixed.